scripts/cli.py

```
@@ -0,0 +1,407 @@
+    """
+    Updates the status of a specified change request and sends a notification about the status change.
+    
+    Parameters:
+        number (str): The unique identifier of the change request to update.
+        status (str): The new status to set for the change request.
+        comments (str): Optional comments to include with the status update.
+    """
+    try:
+        sn_client = ctx.obj['sn_client']
+        
+        change = sn_client.update_change_status(number, status, comments)
```

Ruff (0.11.9)

Local variable `change` is assigned to but never used

Remove assignment to unused variable `change`

(F841)